OpenEdX/Managers/DeepLinkManager/Services/BranchService.swift

```diff
@@ -20,6 +35,7 @@ class BranchService: DeepLinkService {
         open url: URL,
         options: [UIApplication.OpenURLOptionsKey: Any]
     ) -> Bool {
-        false
+        Branch.getInstance().application(app, open: url, options: options)
+        return true
```

Comment:
I think we need to return here result of `Branch.getInstance().application(` function not just `true`. WDYT?